helpers/dbload.ts

```diff
@@ -96,7 +97,8 @@ UserLesson.belongsTo(User, {
   as: 'user'
 })
 
-sequelize.sync({ alter: !!process.env.ALTER_DB })
+// Calling this method causes Sequelize to alter the DB, undoing changes from Prisma migrations
```

Comment:
So this will prevent sequelize from altering DB according to its models in the future.  Should we reapplyl prisma migrations again to fix current state of production db?

Reply:
Not necessarily all the migrations, from what I've seen it's just some foreign keys constraints that we had changed, to set `ON DELETE CASCADE` for example, that sequelize messed up.

I need to look at the production db to see what has been changed. I think writing a new migration is easier too, since prisma already keeps track of applied migrations it might make things too complicated to re-apply.